tests/bulk_create/models.py

```diff
@@ -81,6 +81,9 @@ class NullableFields(models.Model):
     positive_small_integer_field = models.PositiveSmallIntegerField(null=True, default=4)
     small_integer_field = models.SmallIntegerField(null=True, default=5)
     time_field = models.TimeField(null=True, default=timezone.now)
+    auto_field = models.AutoField(null=True, default=6)
+    small_auto_field = models.SmallAutoField(null=True, default=7)
+    big_auto_field = models.BigAutoField(null=True, default=8)
```

Comment:
Model cannot have multiple `AutoField`. For `AutoField` you can use auto created `pk`, for `SmallAutoField` and `BigAutoField` you need to add separate models.

Reply:
Are the tests always run on clean database?

Reply:
Yes.

Reply:
Changed direct `models.AutoField` to `models.ForeignKey to AutoField`